app/mainAppWindow/index.js

```diff
@@ -70,19 +70,20 @@ exports.onAppReady = function onAppReady() {
 	}
 };
 
+let allowFurtherRequests = true;
+
 exports.onAppSecondInstance = function onAppSecondInstance(event, args) {
 	console.log('second-instance started');
-	let allowFurtherRequests = true;
 	if (window) {
 		event.preventDefault();
 		const url = processArgs(args);
 		if (url && allowFurtherRequests) {
 			allowFurtherRequests = false;
-			setTimeout(() => { allowFurtherRequests = true; }, 10000);
-		} else {
-			if (window.isMinimized()) window.restore();
-			window.focus();
+			setTimeout(() => { allowFurtherRequests = true; }, 1000);
```

Comment:
I think the timeout was that big as otherwise meetings can retrigger quite fast making the app reload multiple times. I suggest you increase it at least to 5 seconds (5000).

Reply:
```suggestion
			setTimeout(() => { allowFurtherRequests = true; }, 5000);
```

Reply:
Thanks. Done.